notebooks/en/Prompt_Tuning_PEFT.ipynb

```diff
@@ -0,0 +1,7324 @@
+{
```

Comment:
<p>Using sampling with temperature=0.2 and  top_p=0.95  would result in better generations.</p><pre class="ql-syntax" spellcheck="false">def get_outputs(model, inputs, max_new_tokens=100):
&nbsp; &nbsp; outputs = model.generate(
&nbsp; &nbsp; &nbsp; &nbsp; input_ids=inputs["input_ids"],
&nbsp; &nbsp; &nbsp; &nbsp; attention_mask=inputs["attention_mask"],
&nbsp; &nbsp; &nbsp; &nbsp; max_new_tokens=max_new_tokens,
        temperature=0.2,
        top_p=0.95,
&nbsp; &nbsp; &nbsp; &nbsp; do_sample=True,
&nbsp; &nbsp; &nbsp; &nbsp; repetition_penalty=1.2, #Avoid repetition.
&nbsp; &nbsp; &nbsp; &nbsp; early_stopping=True, #The model can stop before reach the max_length
&nbsp; &nbsp; &nbsp; &nbsp; eos_token_id=tokenizer.eos_token_id
&nbsp; &nbsp; )
&nbsp; &nbsp; return outputs
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Thank you very much! I was aiming for a response as consistent as possible to ensure the explanation in the notebook consistently aligns with the generated answer. With these values, a different response is obtained in each execution.</p><p> </p><p>Would it be acceptable if I leave it as is, with these parameters commented, indicating that for better but more varied responses, users can uncomment and adjust the parameters?</p>

Reply:
<p>I think it's ok to leave as is, and add this context in markdown before the code. It's always great to see the reasoning behind parameter choices, and alternative options. </p>

Reply:
<p>Done</p>